macos/scripts/apply-system-defaults.sh

```
@@ -33,25 +33,25 @@ fi
 
 # 設定ファイルの存在確認と実行
 if [[ ! -f "${SETTINGS_FILE}" ]]; then
-    echo "[WARN] Settings file not found: ${SETTINGS_FILE}"
+    echo "[WARN] System defaults file not found: ${SETTINGS_FILE}"
     echo "[INFO] You can generate it by running 'make backup-defaults'."
 else
-    echo "[INFO] Sourcing settings file: ${SETTINGS_FILE}"
+    echo "[INFO] Sourcing system defaults file: ${SETTINGS_FILE}"
     # `source` を使用して設定を適用するが、エラーが発生しても続行
     if ! source "${SETTINGS_FILE}"; then
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)